installation-bundle/src/Database/Installer.php

```diff
@@ -255,6 +255,10 @@ private function checkEngineAndCollation(array &$sql, array &$order, Schema $fro
             if (strtolower($tableOptions->Engine) !== strtolower($engine)) {
                 if ($innodb && $dynamic) {
                     $command = 'ALTER TABLE '.$tableName.' ENGINE = '.$engine.' ROW_FORMAT = DYNAMIC';
+
+                    if (false !== stripos($tableOptions->Create_options, 'key_block_size=')) {
+                        $command .= ', KEY_BLOCK_SIZE=0';
```

Comment:
```suggestion
                        $command .= ' KEY_BLOCK_SIZE = 0';
```
We should use consistend formatting of the query I guess.